pyomo/contrib/pynumero/cmake/asl_interface/src/AmplInterface.cpp

```diff
@@ -163,6 +160,7 @@ AmplInterface::~AmplInterface() {
     pi0 = NULL;
     delete[] havepi0;
     havepi0 = NULL;
+    delete oi;
```

Comment:
Because this is now a class member, it might be good/safe to follow this with `oi = NULL;`